src/cli/cli.hpp

```diff
@@ -177,8 +177,12 @@ class Interpreter
      * Write an IPv6 address to the CLI console.
      *
      * @param[in]  aAddress  A reference to the IPv6 address.
+     *
+     * @returns The number of bytes placed in the output queue.
+     *
+     * @retval  -1  Driver is broken.
```

Comment:
minor nit to be consistent:
```suggestion
     * @retval  -1  Driver is broken.
     *
```